### PR TITLE
converted desc to sentence case

### PR DESCRIPTION
**Pull request by Dksie09**
## What does this PR do?

Convert descriptions from title case to sentence case

## Test Plan

Manual inspection

## Related PRs and Issues

- (Related PR or issue)

## Checklist

- [x] Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?
- [x] If the PR includes a change to an API's metadata (desc, label, params, etc.), does it also include updated API specs and example docs?

### CODE REVIEW

File: app/controllers/api/account.php [hunk 1]
```diff
@@ -780,7 +780,7 @@
     });
 
 App::put('/v1/account/sessions/magic-url')
-    ->desc('Create Magic URL session (confirmation)')
+    ->desc('Create magic URL session (confirmation)')
```
Comment (gewenyu99) [inline app/controllers/api/account.php:783]: Not related to this PR, @Dksie09 please ignore message below.

What do y'all think about just calling this `Confirm magic URL session`. Like why did we use brackets?

File: app/controllers/api/account.php [hunk 2]
```diff
@@ -1870,7 +1870,7 @@
     });
 
 App::patch('/v1/account/sessions/:sessionId')
-    ->desc('Update OAuth Session (Refresh Tokens)')
+    ->desc('Update OAuth session (refresh tokens)')
```
Comment (gewenyu99) [inline app/controllers/api/account.php:1873]: @Dksie09 ignore this
Ditto this, why did we go with brackets 👀

File: app/controllers/api/databases.php [hunk 3]
```diff
@@ -1517,7 +1517,7 @@ function updateAttribute(
 
 App::post('/v1/databases/:databaseId/collections/:collectionId/attributes/datetime')
     ->alias('/v1/database/collections/:collectionId/attributes/datetime', ['databaseId' => 'default'])
-    ->desc('Create DateTime Attribute')
+    ->desc('Create dateTime attribute')
```
Comment (gewenyu99) [inline app/controllers/api/databases.php:1520]:
```suggestion
    ->desc('Create datetime attribute')
```

File: app/controllers/api/graphql.php [hunk 4]
```diff
@@ -58,7 +58,7 @@
     });
 
 App::post('/v1/graphql/mutation')
-    ->desc('GraphQL Endpoint')
+    ->desc('GraphQL endpoint')
```
Comment (gewenyu99) [inline app/controllers/api/graphql.php:61]: @abnegate Should this have a better desc?

File: app/controllers/api/health.php [hunk 5]
```diff
@@ -12,7 +12,7 @@
 use Utopia\Storage\Storage;
 
 App::get('/v1/health')
-    ->desc('Get HTTP')
+    ->desc('Get http')
```
Comment (gewenyu99) [inline app/controllers/api/health.php:15]:
```suggestion
    ->desc('Get HTTP')
```

File: app/controllers/api/health.php [hunk 6]
```diff
@@ -47,7 +47,7 @@
     });
 
 App::get('/v1/health/db')
-    ->desc('Get DB')
+    ->desc('Get db')
```
Comment (gewenyu99) [inline app/controllers/api/health.php:50]:
```suggestion
    ->desc('Get DB')
```

File: app/controllers/api/graphql.php (hunk 4 above)
Comment (abnegate) [reply app/controllers/api/graphql.php:61]: Yep we could add some more detail here, could be particularly useful since it ends up in SDK docblocks

Comment (gewenyu99) [reply app/controllers/api/graphql.php:61]: I've added a task for this. We'll address these separately to avoid scope creep.

**Review by eldadfux — APPROVED** [review]

File: app/controllers/api/teams.php
Comment (Thelastjoke) [inline app/controllers/api/teams.php:1]: @Dksie09 

> `
IMG_20231003_075205
`

**Pull request merged**